nexus-repository-conan/src/main/java/org/sonatype/repository/conan/internal/orient/proxy/v1/ConanProxyControllerV1.java

```diff
@@ -10,14 +10,15 @@
  * of Sonatype, Inc. Apache Maven is a trademark of the Apache Software Foundation. M2eclipse is a trademark of the
  * Eclipse Foundation. All other trademarks are the property of their respective owners.
  */
-package org.sonatype.repository.conan.internal.proxy.v1;
+package org.sonatype.repository.conan.internal.orient.proxy.v1;
 
 import javax.inject.Inject;
 import javax.inject.Named;
 import javax.inject.Singleton;
 import org.sonatype.nexus.repository.view.Router;
-import org.sonatype.repository.conan.internal.common.v1.ConanControllerV1;
+import org.sonatype.repository.conan.internal.orient.common.v1.ConanControllerV1;
 import org.sonatype.repository.conan.internal.common.v1.ConanRoutes;
+import org.sonatype.repository.conan.internal.proxy.v1.ConanProxyHandlers;
```

Comment:
Should `CoanProxyHandlers` also be moved? The hosted equivalent one was.

Reply:
There is no dependency on orient there, but i'm assuming when the newdb implementation is done things will shift around a bit anyway

Reply:
> There is no dependency on orient there, but i'm assuming when the newdb implementation is done things will shift around a bit anyway

Unless I'm missing something it has the same level of dependency as `HostedHandlers`